src/Connector.SqlServer/Utility/SanitizedSqlString.cs

```diff
@@ -0,0 +1,47 @@
+﻿using CluedIn.Connector.Common.Helpers;
+using System;
+
+namespace CluedIn.Connector.SqlServer.Utility
+{
+    public class SanitizedSqlString
```

Comment:
Motivation behind: to do sanitization only once, and know afterwards that string was sanitized.